### PR TITLE
Adds light mode

### PR DESCRIPTION
**Pull request by mouradaouinat**
This is a work in progress, there are some of the SVGS that still need to be converted, Also need your opinion on the approach

### CODE REVIEW

**Comment by aboullaite** [comment]
It looks very nice! Thanks for such great effort

**Comment by mouradaouinat** [comment]
Part of the credit goes also to @ismailElazizi

File: src/components/Home/Hero/index.js [hunk 1]
```diff
@@ -24,8 +28,14 @@ export default () => {
           </Link> */}
         </div>
       </div>
-      <HeroImageMobile className="crea mobile-only" />
-      <HeroImage className="crea desktop-only" />
+      {theme === "dark" ? (
```
Comment (yjose) [inline src/components/Home/Hero/index.js:31]: Not sure why, but look like hero svgs have different size

**Comment by mouradaouinat** [comment]
Thanks! Sounds legit, I'm on it 😃

**Comment by yjose** [comment]
Also, you need to update the footer color , check https://geeksblabla.com/

**Comment by mouradaouinat** [comment]
Only two SVGs to go now @ismailElazizi 🙌

**Comment by yjose** [comment]
@ismailElazizi  can you check 👆 , please

**Comment by aboullaite** [comment]
@mouradaouinat 🙌👍

**Comment by midrissi** [comment]
@mouradaouinat 👍 👏

**Pull request merged**